monai/transforms/transforms.py

```diff
@@ -72,12 +72,12 @@ class Flip:
     """
 
     def __init__(self, axes=None):
```

Comment:
Why you allow a default value of **axes**=None?
I think we must ask the user to set axes, do you agree?
Thanks.

Reply:
By default, it flips over all the axes.